torch_ort/docker/Dockerfile.onnxruntime-torch2.0.0-cu118-cudnn8-devel-ubuntu20.04

```diff
@@ -34,10 +43,13 @@ RUN pip install --upgrade pip
 
 # PyTorch
 RUN pip install onnx ninja
-RUN pip install torch==${TORCH_VERSION}+${TORCH_CUDA_VERSION} torchvision==${TORCHVISION_VERSION} -f https://download.pytorch.org/whl/torch/
+RUN pip install torch==${TORCH_VERSION} torchvision==${TORCHVISION_VERSION} -f https://download.pytorch.org/whl/${TORCH_CUDA_VERSION}
```

Comment:
Seems like this is using `/usr/bin/pip`? Not sure if we want to use miniconda or /usr/bin/pip.

I think we could use miniconda over /usr/bin/pip.

Maybe this can happen later though.

Reply:
No, this should be using miniconda. miniconda is higher up in PATH.

Reply:
I am not sure, but my guess is that although miniconda is higher up in PATH, the new PATH has not been loaded yet. So, miniconda pip is not in use yet.

Reply:
[Here](https://github.com/microsoft/onnxruntime/blob/main/tools/ci_build/github/linux/docker/scripts/install_python_deps.sh#L31-L36) is what we do in `onnxruntime`.

Reply:
ok let me test and fix this